docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Code Reviews
+
+Code reviews are intended to help all of us grow as engineers and improve the quality of what we ship.
+These guidelines are meant to reinforce those two goals.
+
+## For reviewers
+
+Aim to respond to code reviews within 1 business day.
+
+Remember to highlight things that you like and appreciate while reading through the changes,
+and to make any other feedback clearly actionable by indicating if it is optional preference, an important consideration, or an error.
+
+Don't be afraid to comment with a question, or to ask for clarification, or provide a suggestion,
+whenever you don’t understand what is going on at first glance — or if you think an approach or decision can be improved.
+Code reviews give us a chance to learn from one another, and to reflect, iterate on, and document why certain decisions are made.
+
+Once you're ready to approve or request changes, err on the side of trust.
+Send a vote of approval if the PR looks ready except for small minor changes,
+and trust that the recipient will address your comments before merging by replying via comment or code to any asks.
+Use "request changes" sparingly, unless there's a blocking issue or major refactors that should be done.
+
+## For authors or requesters
+
+### Draft PRs
+
+If your PR is a work-in-progress, or if you are looking for specific feedback on things,
+create a [Draft Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests)
+and state what you are looking for in the description.
+
+Your PR should be small enough that a reviewer can reasonably respond within 1-2 business days.
+For larger changes, break them down into a series of PRs.
+If refactors are included in your changes, try to split them out as recommended below.
```

Comment:
Where is the recommendation this refers to?

Reply:
looked for a while and can't find it, even in the original docs that this was based off of, so i'm going to remove the "as recommended below"